app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantsTiles.kt

```diff
@@ -128,8 +131,8 @@ fun VerticalCallingPager(
                     }
                 }
             }
-            // we don't need to display the indicator if we have one page
-            if (pagesCount(participants.size) > 1) {
+            // we don't need to display the indicator if we have one page and when it's in PiP mode
+            if (pagesCount(participants.size) > 1 && !activity.isInPictureInPictureMode) {
```

Comment:
Question on this:

If user is in PiP mode and we do have more than one page, then the page indicators will not show? 🤔

Reply:
Exactly, as in PiP user can not interact with that small view.